spaceship/lib/spaceship/connect_api/models/app.rb

```
@@ -0,0 +1,508 @@
+        resps = client.get_available_territories(app_id: id, filter: filter, includes: includes, limit: limit, sort: sort).all_pages
+        return resps.flat_map(&:to_models)
+      end
+
+      #
+      # App Pricing
+      #
+
+      def fetch_app_prices(client: nil, filter: {}, includes: "priceTier", limit: nil, sort: nil)
+        client ||= Spaceship::ConnectAPI
+        resp = client.get_app_prices(app_id: id, filter: filter, includes: includes, limit: limit, sort: sort)
+        return resp.to_models
```

RuboCop (1.89.0)

[convention] Redundant `return` detected.

(Style/RedundantReturn)